src/plugins/usage_collection/README.mdx

```diff
@@ -7,6 +7,8 @@ date: 2021-02-24
 tags: ['kibana','dev', 'contributor', 'api docs']
 ---
 
+IMPORTANT: Usage collection and telemetry applies to Kibana developers only.
```

Comment:
Super-NIT: Maybe it's only a wording thingy: should we say _... internal Kibana developers..._? Maybe external developers also call themselves "Kibana developers" (like we do with React) 😇

Reply:
How about "Usage collection and telemetry applies to internal Elastic Kibana developers only."
I've also realized that the Plugin List generator [grabs the first sentence](https://github.com/elastic/kibana/pull/95121/files#diff-ad53fc4d83ae8ebe6554cf17205d0945bb79f4bea23a137909b9baac20b53a97R234) from the README as a description to what the plugin does. Are we ok with that or should we move the `WARNING` to within the opening paragraph?

Reply:
> should we move the `WARNING` to within the opening paragraph?

I think adding it to the opening paragraph makes more sense? At least there's a bit of an explanation of what's the plugin about?

Reply:
Done in [`47e9db1` (#95121)](https://github.com/elastic/kibana/pull/95121/commits/47e9db14579f3f13b51549e2068d9af0ebe94a08) and another whitespace fix in [`47e9db1` (#95121)](https://github.com/elastic/kibana/pull/95121/commits/47e9db14579f3f13b51549e2068d9af0ebe94a08).
FWIW, the only way I can get around the whitespace issues is pushing the changes made locally and then fixing the whitespace directly through Github.